packages/gatsby-theme-docs/gatsby-node.js

```diff
@@ -28,8 +29,44 @@ exports.onPreBootstrap = ({ reporter }) => {
       fs.mkdirSync(dir);
     }
   });
+
+  createReleaseNoteIndex(reporter);
 };
 
+function createReleaseNoteIndex(reporter) {
+  const releaseNoteIndex = 'src/releases/index.mdx';
+  if (!fs.existsSync(releaseNoteIndex)) {
+    fs.writeFile(
+      releaseNoteIndex,
+      `---
+date: 1970-01-01 # necessary in this format
+title: (title goes here)
+description: |
+  (description for the template)
+type: enhancement # feature | fix | enhancement
+topics:
+  - foo
+  - barr
+  # Topics is an array, here are some examples:
+  #  - Products
+  #  - Product List
+  #  - Categories
+  #  - Carts
+  #  - API Clients
+  #  - Settings
+---
+      `,
+      (error) => {
+        if (error) {
+          throw error;
+        }
+
+        reporter.info(`Successfully created "${releaseNoteIndex}"`);
+      }
+    );
+  }
+}
+
```

Comment:
What is this? 🧐

Reply:
Build fails for other micro-sites because release notes fields are not available to be queried, this just creates a file that provides the fields in graphql and as a release note template.

Reply:
Sorry but this is not the right approach.

First of all, release notes are optional for a website. Therefore we need to deal with that and do not assume that there will always be release notes.
For instance, in the my other PR I added a configuration option `hasReleaseNotes` that can be configured for the theme: https://github.com/commercetools/commercetools-docs-kit/pull/336/files#diff-afcde1e6d3112cafc056b077adde712fR73

Based on this flag, we should then setup and render the release notes related stuff or not.

Furthermore, if we need to ensure that fields/nodes are defined (in case they are optional) we need to tell gatsby about it. For example like we do with the navigation yaml: https://github.com/commercetools/commercetools-docs-kit/blob/e18623745a28e85c23581836a0d073f50cbbcf1a/packages/gatsby-theme-docs/gatsby-node.js#L33-L69

Reply:
Why not define contents field in the same way you do with navigation yaml? Why does it have to be unique to release notes?